src/pages/information/agenda/index.jsx

```diff
@@ -0,0 +1,148 @@
+        </SelectableTableRow>
+    )
+}
+
+export const AgendaList = (props) => {
+    const [activeContent, setActiveContent] = useState(1);
+    const [currentEvent, setCurrentEvent] = useState(undefined);
+
+    const [agendaList, setAgendaList] = useState([]);
+
+    const [ window, setWindow ] = useState([]);    
+    const [loading, setLoading] = useState(true);
+
+    const reloadAgendaList = async () => {
+        setCurrentEvent(await getCurrentEvent());
+        const getAgendaList = await Agenda.getAgenda();
+        if (getAgendaList) {
+            setAgendaList(getAgendaList);
+            setLoading(false);
+        }
+      };
+    
+    useEffect(async () => {
+        reloadAgendaList().catch((e) => {
+            console.log(e);
+        })
+    }, []);
+
+    const reload = async () => {
+        await reloadAgendaList();
+    }
+
+    if(loading) {
+        return (
+            <PageLoading />
+        )
+    } else {
+        return (
+            
+            <>
+                {window}
+                <DashboardHeader>
+                    <DashboardTitle>
+                        Program
+                    </DashboardTitle>
+                </DashboardHeader>
+
+                <DashboardBarSelector border>
+                    <DashboardBarElement active={activeContent == 1} onClick={() => setActiveContent(1)}>Oppsett</DashboardBarElement>
+                </DashboardBarSelector>
+
+                <DashboardContent visible={activeContent == 1}>
+                    <InnerContainer>
+                        Programmet er hva som skal skje under arrangementet.<br />
+                        Legg til dato og tid, tittel og beskrivelse for hva som skal skje.<br /><br />
+
+                        Informasjonen du legger inn vises på hovedsiden under tidsplan og på infosiden/infoskjermene som blir satt opp på lokasjonen.
+                    </InnerContainer>
+
+                    <InnerContainerRow nopadding>
+                        <InnerContainer flex="1">
+                            <PanelButton onClick={() => setWindow(newWindow({title: "Opprett nytt element", subtitle: currentEvent.name, Component: NewAgendaEntry, exitFunction: () => {setWindow(false); reload()}}))} icon={faPlus}>Legg til</PanelButton>
+                        </InnerContainer>
+                    </InnerContainerRow>
+
+                    <InnerContainer>
+                        <Table>
+                            <TableHead border>
+                                <TableRow>
+                                    <TableCell as="th" center   flex="0 1.3rem" mobileHide          title="Indikerer om elementet er synlig på infoskjermene eller ikke."><InnerColumnCenter>...</InnerColumnCenter></TableCell>
```

Comment:
This styling would be removed by any sane formatter.

Reply:
What could it look like instead?

Reply:
Normal - one space between each :D

Reply:
Ahaa, ok, got it!